utils/workflow.go

```diff
@@ -21,52 +20,56 @@ import (
 	"encoding/gob"
 	"encoding/json"
 	"errors"
-	apiv08 "github.com/kiegroup/kogito-serverless-operator/api/v1alpha08"
-	"github.com/kiegroup/kogito-serverless-operator/constants"
-	"github.com/kiegroup/kogito-serverless-operator/converters"
+
+	"k8s.io/apimachinery/pkg/util/runtime"
 	ctrllog "sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/kiegroup/kogito-serverless-operator/api/metadata"
+
+	operatorapi "github.com/kiegroup/kogito-serverless-operator/api/v1alpha08"
 )
 
-// GetWorkflowFromCR return a Kogito compliant workflow as bytearray give a specific workflow CR
-func GetWorkflowFromCR(workflowCR *apiv08.KogitoServerlessWorkflow, ctx context.Context) ([]byte, error) {
-	log := ctrllog.FromContext(ctx)
-	converter := converters.NewKogitoServerlessWorkflowConverter(ctx)
-	workflow, err := converter.ToCNCFWorkflow(workflowCR)
+const defaultImageTag = ":latest"
```

Comment:
image tag should be `latest`.
the `:` could lead to issues IMO

Reply:
This is just a renaming, I'd rather not touch this feature in this PR.

Reply:
fair enough